src/gap.c

```diff
@@ -1122,32 +1122,32 @@ static int SetExitValue(Obj code)
 
 /****************************************************************************
 **
-*F  FuncGAP_EXIT_CODE() . . . . . . . . Set the code with which GAP exits.
+*F  FuncGapExitCode() . . . . . . . . Set the code with which GAP exits.
 **
 */
 
-static Obj FuncGAP_EXIT_CODE(Obj self, Obj code)
+static Obj FuncGapExitCode(Obj self, Obj code)
 {
   if (!SetExitValue(code))
-    ErrorQuit("GAP_EXIT_CODE: Argument must be boolean or integer", 0, 0);
+    ErrorQuit("GapExitCode: Argument must be boolean or integer", 0, 0);
   return (Obj) 0;
 }
 
 
 /****************************************************************************
 **
-*F  FuncQUIT_GAP()
+*F  FuncQuitGap()
 **
 */
 
-static Obj FuncQUIT_GAP(Obj self, Obj args)
+static Obj FuncQuitGap(Obj self, Obj args)
 {
   if ( LEN_LIST(args) == 0 ) {
     SystemErrorCode = 0;
   }
   else if ( LEN_LIST(args) != 1 
             || !SetExitValue(ELM_PLIST(args, 1) ) ) {
-    ErrorQuit( "usage: QUIT_GAP( [ <return value> ] )", 0, 0);
+    ErrorQuit( "usage: QuitGap( [ <return value> ] )", 0, 0);
```

Comment:
BTW, I am not aware of any code calling `QuitGap` or `ForceQuitGap` without an exit code. Are you? What would be a use case?

Reply:
I don't think I've done it myself. I suppose someone might want to just quit gap, but asking for a return value doesn't feel unreasonable.